src/storage/postgres/src/main/java/org/locationtech/geogig/storage/postgresql/PGGraphDatabase.java

```diff
@@ -198,11 +201,13 @@ public void truncate() {
             protected Void doRun(Connection cx) throws IOException, SQLException {
                 cx.setAutoCommit(false);
                 try {
-                    cx.createStatement().execute(format("DELETE FROM %s", MAPPINGS));
-                    cx.createStatement().execute(format("DELETE FROM %s", PROPS));
-                    cx.createStatement().execute(format("DELETE FROM %s", EDGES));
-                    cx.createStatement().execute(format("DELETE FROM %s", NODES));
-                    cx.commit();
+                    try (Statement st = cx.createStatement()) {
+                        st.execute(log(format("TRUNCATE %s", MAPPINGS), LOG));
+                        st.execute(log(format("TRUNCATE %s", PROPS), LOG));
+                        st.execute(log(format("TRUNCATE %s", EDGES), LOG));
+                        // this is a view now. st.execute(format("DELETE FROM %s", NODES));
+                        cx.commit();
```

Comment:
This will delete the graphs for every repo in the database.  Some logic should be added to not break the other repos.